docs/troubleshooting.md

```diff
@@ -1,32 +1,54 @@
 # Troubleshooting
 
-- [Installation](#installation)
-  - [How to use LISA in WSL](#how-to-use-lisa-in-wsl)
-  - [Cannot find package after run `poetry install`](#cannot-find-package-after-run-poetry-install)
-  - [Error: Poetry could not find a pyproject.toml file](#error-poetry-could-not-find-a-pyprojecttoml-file)
-  - [Poetry related questions](#poetry-related-questions)
+- [Troubleshooting](#troubleshooting)
+  - [Installation](#installation)
+    - [How to use LISA in WSL](#how-to-use-lisa-in-wsl)
+    - [Cannot find package after run `poetry install`](#cannot-find-package-after-run-poetry-install)
+    - [Error: Poetry could not find a pyproject.toml file](#error-poetry-could-not-find-a-pyprojecttoml-file)
+  - [Using VSCode](#using-vscode)
+    - [Cannot find Python Interpreter by Poetry](#cannot-find-python-interpreter-by-poetry)
+    - [VSCode Python extension no longer supports `python.pythonPath` in `setting.json`](#vscode-python-extension-no-longer-supports-pythonpythonpath-in-settingjson)
   - [Other issues](#other-issues)
+    - [Poetry related questions](#poetry-related-questions)
 
 ## Installation
 
 ### How to use LISA in WSL
 
-If you are using WSL, installing Poetry on both Windows and WSL may cause both platforms' versions of Poetry to be on your path, as Windows binaries are mapped into `PATH` of WSL. This means that the WSL `poetry` binary _must_ appear in your `PATH` before the Windows version, or this error will appear:
+If you are using WSL, installing Poetry on both Windows and WSL may cause both platforms' versions of Poetry to be on your path, as Windows binaries are mapped into `PATH` of WSL. This means that the WSL `poetry` binary _must_ appear in your `PATH` before the Windows version, otherwise this error will appear:
 
 > `/usr/bin/env: ‘python\r’: No such file or directory`
 
 ### Cannot find package after run `poetry install`
 
-Poetry is case sensitive. When in windows, make sure the case of path is consistent every time.
+Poetry is case sensitive when reading the path. Please make sure the path is consistent all the time and there's no spelling errors or case mismatch.
 
 ### Error: Poetry could not find a pyproject.toml file
 
 Poetry provides different packages according to the folder, and depends on the `pyproject.toml` file in the current folder. Make sure to run `poetry` in the root folder of LISA.
 
-### Poetry related questions
+## Using VSCode
+
+### Cannot find Python Interpreter by Poetry
+
+In the root folder of LISA, run the command below. It will return the path of the virtual environment that Poetry set up. Use that path to find the Python interpreter accordingly (in most cases open the path and look for `\Scripts\python.exe`).
+
+```powershell
+poetry env info -p
+```
+
+### VSCode Python extension no longer supports `python.pythonPath` in `setting.json`
```

Comment:
What's reason to have this item? If so, feel free to remove `python.pythonPath` from our example of `settings.json`.

Reply:
It's just me run into this problem and I wonder if others would as well. I think removing the `python.pythonPath` from the example of `setting.json` could also work, as long as we ensure the user select the right python interpreter.

Reply:
yes, we can ask user to select right python interpreter. You problem is coming from the "python.pythonPath", which is added by me. It's great to see we have another simple way to handle it.